propolis/src/hw/bhyve/atpic.rs

```diff
@@ -0,0 +1,35 @@
+use std::sync::Arc;
+
+use crate::dispatch::DispCtx;
+use crate::inventory::Entity;
+use crate::migrate::Migrate;
+
+use erased_serde::Serialize;
+
+pub struct BhyveAtPic {}
```

Comment:
Are the `atpic` devices even necessary for the types of VMs we want to run? My understanding was that atpic is for managing the pair of 8259 chips in legacy systems.

Reply:
It's in the kernel emulation and not really disabled, so yeah, we need it still (sadly).